### DISCUSSION TITLE
Custom error page

### DISCUSSION THREAD

**Discussion by davetapley** [falconry/falcon#2215] [Q&A] [answered]
How do I customize the error pages such as this?
I see [`add_error_handler`](https://falcon.readthedocs.io/en/3.1.3/api/api.html#falcon.API.add_error_handler), but it doesn't mention anything about how the actual response is rendered 🤔 

<img width="784" alt="Screen Shot 2024-03-15 at 3 33 36 PM" src="https://github.com/falconry/falcon/assets/48232/67fe2505-d950-479f-852c-7b28d48eb9e7">

**Comment by vytas7** [comment, marked as answer]
Hi @davetapley!
Yeah, this area of our docs is unfortunately a bit lacking. I think we even have a docs issue tracking customization of error handling in particular.

You can also check if you could find anything useful in my Gist: https://gist.github.com/vytas7/068ac5bd8a0fd118052d32e30be67792

**Reply by davetapley** [reply]
Yep, https://github.com/falconry/falcon/issues/1460 looks like the one.
I have a question, but I'll ask on the gist. Thanks!
